inc/functions_post.php

```diff
@@ -401,64 +401,81 @@ function build_postbit($post, $post_type=0)
 		}
 
 		// Display profile fields on posts - only if field is filled in
-		if(is_array($profile_fields))
```

Comment:
> Probably adding your if(!isset($profile_fields))check before the current if(is_array($profile_fields)) one would fix the quick reply. That would be another issue.

How should that fix the issue? `is_array` returns false if the variable isn't set anyways.

Anyways, @Sama34 are you okay with merging this and fixing the rest in another PR in an upcoming version?